methods/incidentReports.coffee

```diff
@@ -35,18 +47,20 @@ Meteor.methods
       Meteor.call("editUserEventLastIncidentDate", incident.userEventId)
     return incident._id
 
-  editIncidentReport: (incident) ->
+  editIncidentReport: (incident, userEventId) ->
```

Comment:
Question: do you think we should keep `addIncidentReportToEvent` in the "incidentReports" file or should it be with userEvent methods?